content/docs/visualization/template_examples.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Template examples
+sort_rank: 4
+---
+
+# Template examples
+
+Prometheus supports templating in the summary and description fields of
+alerts, as well as in served console pages. Templates have the ability to run
+queries against the local database, iterate over data, use conditionals, format
+data, etc. This Prometheus templating language is based on the 
```

Comment:
"This" sounds a bit weird here with the extra specification of "Prometheus" after the "This", no? How about just "The"?